### PR TITLE
CMD2-123

### PR DESCRIPTION
**Pull request by yowmamasita**
`((?<!([A-Z]{1,10})-?)[A-Z]+\d?-\d+)`

IAM-123

### CODE REVIEW

File: .github/mergeable.yml [hunk 1]
```diff
@@ -8,6 +8,6 @@ mergeable:
     validate:
       - do: description
         must_include:
-          regex: '((?<!([A-Z]{1,10})-?)[A-Z]+-\d+)'
+          regex: '((?<!([A-Z]{1,10})-?)[A-Z]+\d?-\d+)'
```
Comment (dmussaku) [inline .github/mergeable.yml:11]:
```suggestion
          regex: '([A-Z]{2,10})+\d?-\d+'
```

Comment (yowmamasita) [reply .github/mergeable.yml:11]: the lookbehind is for not matching `ABC-123-456`

Comment (dmussaku) [reply .github/mergeable.yml:11]: Screenshot 2023-02-17 at 16 47 22
Screenshot 2023-02-17 at 16 47 10

Looks the same no?

Comment (dmussaku) [reply .github/mergeable.yml:11]: Anyways, I'll approve and merge for now will see if needs fixing later.

**Review by dmussaku — APPROVED** [review]

File: .github/mergeable.yml (hunk 1 above)
Comment (yowmamasita) [reply .github/mergeable.yml:11]: you're right. might be that the lookbehind is not supported anymore

**Pull request merged**